src/components/Account-panel/Change-password/Change-password.tsx

```diff
@@ -1,3 +1,141 @@
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import axiosInstance from '../../../axios/settings'
+import {
+  Box,
+  Button,
+  FormHelperText,
+  InputLabel,
+  OutlinedInput,
+  Typography
+} from '@mui/material'
```

Comment:
we already have some components as a generic with mui theme, use them instead of default

Reply:
done